css/style.css

```diff
@@ -0,0 +1,141 @@
+:root {
+    --color-light: oldlace;
+    --color-dark: #333;
+    --color-primary: rgb(148, 226, 148);
+    --color-primary-dark: rgb(53, 73, 53);
+}
+ 
+ * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Setting the value of 1rem */
+html {
+    /* 1rem = 10 pixels */
+    font-size: 62.5%;
+}
```

Comment:
Please clean up unused code in order to organize and clarify the coding.

Reply:
- [ ] What do think if you improve :chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive, politely offer a suggestion 😏, and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖. I know you can do it :muscle: :muscle: